stable/kong/values.yaml

```diff
@@ -142,3 +142,33 @@ postgresql:
   postgresDatabase: kong
   persistence:
     enabled: false
+
+# Kong Ingress Controller's primary purpose is to satisfy Ingress resources
+# created in k8s.  It uses CRDs for more fine grained control over routing and
+# for Kong specific configuration.
+ingressController:
+  enabled: false
+  image:
+    repository: kong-docker-kubernetes-ingress-controller.bintray.io/kong-ingress-controller
+    tag: 0.1.0
+  replicaCount: 1
+  livenessProbe:
+    failureThreshold: 3
+    httpGet:
+      path: "/healthz"
+      port: 10254
+      scheme: HTTP
+    initialDelaySeconds: 180
```

Comment:
@shashiranjan84 Thoughts on these delays?

Reply:
I don't think its needs to be that high. Also I would increase timeoutSeconds to 5 or 10